File: CS50 X/problem set 1/mario/mario.c
#include<cs50.h>
#include<stdio.h>

int main(void)
{
    int hight=0;
    do
    {
        hight=get_int("Hight: ");
    }
    while(hight<=0);
    if (hight)
    {
        for(int i=0;i<hight;i++)
        {
            for(int j=1;j<hight-i;j++)
            {
                printf(" ");
            }
            for(int f=0;f<i+1;f++)
            {
                printf("#");
            }
            printf("\n");
        }
    }
    return 0;
}

File: CS50 X/problem set 6/mario/mario.py
from cs50 import get_int
while True:
    hight=get_int(("Hight:"))
    if (hight<1 or hight>8):
        continue
    for line in range(1,hight+1):
        print(" "*(hight-line),end='')
        print("#"*line,end='')
        print()
    break
